src/mmw/package.json

```diff
@@ -49,6 +49,8 @@
     "turf-random": "1.0.2",
     "underscore": "1.8.3",
     "watchify": "3.1.0",
-    "zeroclipboard": "2.2.0"
+    "zeroclipboard": "2.2.0",
+    "turf-intersect": "1.4.2",
```

Comment:
Did you run it locally or via `npm.sh` in the VM? When I run it in the VM I get the sorted version (although with carets in the version numbers which should be removed):

```bash
$ ./scripts/npm.sh install turf-erase turf-intersect --save
+ ARGS='install turf-erase turf-intersect --save'
+ vagrant ssh app -c 'cd /opt/app && envdir /etc/mmw.d/env npm install turf-erase turf-intersect --save'
npm WARN package.json model-my-watershed@0.0.1 No description
npm WARN package.json model-my-watershed@0.0.1 No README data
npm WARN package.json iframe-phone@1.1.3 No repository field.
npm WARN package.json retina.js@1.3.0 No repository field.
turf-intersect@1.4.2 node_modules/turf-intersect
└── jsts@0.15.0 (javascript.util@0.12.5)

turf-erase@1.3.2 node_modules/turf-erase
├── turf-featurecollection@1.0.1
└── jsts@0.15.0 (javascript.util@0.12.5)
Connection to 127.0.0.1 closed.
```

```shell
$ git diff
```

```diff
diff --git a/src/mmw/package.json b/src/mmw/package.json
index f7ace2d..3100bca 100644
--- a/src/mmw/package.json
+++ b/src/mmw/package.json
@@ -32,8 +32,8 @@
     "jstify": "0.9.0",
     "leaflet": "0.7.3",
     "leaflet-draw": "git://github.com/michaelguild13/Leaflet.draw#e2fe3a4",
-    "leaflet.locatecontrol": "0.43.0",
     "leaflet-plugins": "git://github.com/azavea/leaflet-plugins#feature/browserify",
+    "leaflet.locatecontrol": "0.43.0",
     "livereloadify": "2.0.0",
     "lodash": "3.6.0",
     "minifyify": "7.0.0",
@@ -46,11 +46,11 @@
     "testem": "0.5.15",
     "turf-area": "1.1.1",
     "turf-buffer": "1.0.4",
+    "turf-erase": "^1.3.2",
+    "turf-intersect": "^1.4.2",
     "turf-random": "1.0.2",
     "underscore": "1.8.3",
     "watchify": "3.1.0",
-    "zeroclipboard": "2.2.0",
-    "turf-intersect": "1.4.2",
-    "turf-erase": "1.3.2"
+    "zeroclipboard": "2.2.0"
   }
 }
```